MvvmCross.Android.Support/V7.RecyclerView/MvxRecyclerAdapter.cs

```diff
@@ -241,15 +241,10 @@ protected virtual void SetItemsSource(IEnumerable value)
 
         protected virtual void OnItemsSourceCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
         {
-            if (Looper.MainLooper == Looper.MyLooper())
-            {
-                NotifyDataSetChanged(e);
-            }
-            else
-            {
-                var h = new Handler(Looper.MainLooper);
-                h.Post(() => NotifyDataSetChanged(e));
-            }
+            if (Looper.MainLooper != Looper.MyLooper())
+                MvxBindingLog.Error("All collection changes ");
```

Comment:
Is there something missing from the message here?

Reply:
fixed